cmd/executor/x/main.go

```diff
@@ -0,0 +1,223 @@
+package main
```

Comment:
As discussed, I think the `exec` name would be a bit more verbose 🤔 for example I thought `x` is just a temporary name as we don't have anything else and that it doesn't have any specific meaning 😅 


But yeah, `x` can be a default alias for it.

Reply:
I have 2 more PRs based on this one, so if that's ok, I will rename it in the follow-up 👍